canonical/cos-proxy-operator
# Upkeep

## Issue
General upkeep.


## Solution
* Set workload verion
* Fix CI
* Consolidate CI
* Update charm libs


## Testing Instructions
Deploy charm and ensure it is generally functioning correctly (except known bugs)


## Release Notes
* cos-proxy-operator now sets workload version
* cos-proxy-operator update charm libraries
